### PR TITLE
pkg/linksharing/web: design updates v2

### PR DESCRIPTION
**Pull request by NikolaiYurchenko**
Bold test stylings removed from sidebar and footer
Expander appearance changed
Pieces count block and text stylings changed
Footer text centered on mobile and tablet resolutions

### CODE REVIEW

**Review by amwolff — CHANGES_REQUESTED** [review]
Would you mind giving a brief overview of what this pull request does? :) Is that Change-Id related to any Gerrit change? I can't find it, unfortunately.

File: pkg/linksharing/web/static/css/style.css [hunk 1]
```diff
@@ -116,19 +116,33 @@ p, span, a {
...
-  width: 20px;
-  height: 20px;
+.expander,
+.collapser {
+  width: 26px;
+  height: 26px;
   cursor: pointer;
+  position: absolute;
+  top: 50vh;
+  transform: translateY(-50%);
```
Comment (egonelbre) [inline pkg/linksharing/web/static/css/style.css:132]: This need for translateY looks weird. Any reason you are not using `top: calc(50vh - 13px)`?

Or better yet flexbox for aligning things?

**Comment by egonelbre** [comment]
> Is that Change-Id related to any Gerrit change? I can't find it, unfortunately.

Gerrit hooks add change id-s automatically.

File: pkg/linksharing/web/static/css/style.css (hunk 1 above)
Comment (NikolaiYurchenko) [reply pkg/linksharing/web/static/css/style.css:132]: transform: translateY(-50%) needs to not hardcode height/2, so if expander block parameters will be changed we will not change css in 2 places.

Comment (egonelbre) [reply pkg/linksharing/web/static/css/style.css:132]: And flexbox?

Comment (NikolaiYurchenko) [reply pkg/linksharing/web/static/css/style.css:132]: can not see any point for creating 2 more containers to use flexbox for centering 1 image

Comment (NikolaiYurchenko) [reply pkg/linksharing/web/static/css/style.css:132]: it will also cause padding problems

File: pkg/linksharing/web/single-object.html [hunk 2]
```diff
@@ -54,9 +53,10 @@ <h3 class="mb-3">You’re getting this file from all over the world.</h3>
           </div>
         </div>
       </div>
+      <p><img class="collapser" src="{{.Base}}/static/img/collapse.svg" onclick="toggleSideBar()" alt="collapse icon"></p>
     </div>
-
   </div>
+  <p><img class="expander" src="{{.Base}}/static/img/expand.svg" onclick="toggleSideBar()" alt="collapse icon"></p>
```
Comment (amwolff) [inline pkg/linksharing/web/single-object.html:59]: Is alt here correct?

```suggestion
  <p><img class="expander" src="{{.Base}}/static/img/expand.svg" onclick="toggleSideBar()" alt="expand icon"></p>
```

**Review by amwolff — APPROVED** [review]

**Review by VitaliiShpital — APPROVED** [review]

**Pull request merged**